website/project/views/node.py

```diff
@@ -605,6 +606,21 @@ def remove_private_link(*args, **kwargs):
         link = PrivateLink.load(link_id)
         link.is_deleted = True
         link.save()
+
+        for node in link.nodes.all():
```

Comment:
As for the django-y ness, normally I'd be worried about this here, but I do need to add a log to each node, so can't use something like `values` or `values_list` I don't think....